src/modules/message.js

```diff
@@ -0,0 +1,36 @@
+const deepFreeze = require('deep-freeze');
+/**
+  @class Message
+  Convert the given parameters into a frozen (@see Object.deepFreeze()) message package
+  @param logLevel {string} - a message log level (@see ChainLink @class for additional info)
+  @param message {Object|Error} - a message payload. Either a text string or an Error object
+  @param metas {Object} - metadata
+**/
+class Message {
+  constructor(logLevel, message, ...metas) {
+    logLevel = logLevel || process.env.DEFAULT_LOG_LEVEL || 'info'; //eslint-disable-line
```

Comment:
you could move that to line 14, thus use it where you need it and you will escape logLevel overwrite and won't need to disable eslint